src/bot.js

```diff
@@ -20,10 +21,18 @@ bot.on("polling_error", (m) => {
 bot.on("message", onMessage);
 
 async function onMessage(msg) {
+  const userName = msg.from.first_name + " " + msg.from.last_name;
```

Comment:
use user id as identifier, because name + surname is not unique

Reply:
I used the first and last name only to display them in the result table. But selection is carried out only by user id